### ISSUE TITLE
[Rendering] Accessibility: Screen reader cannot read "alternate text" from image with selection action (`Action.OpenUrl`)

### ISSUE THREAD

**Issue by compulim** [microsoft/AdaptiveCards#6442]
### Target Platforms

NodeJS

### SDK Version

2.9.0

### Application Name

Web Chat

### Problem Description

When adding an image element with selection action of type `Action.OpenUrl`, the "alternate text" from the image element cannot be read by the screen reader.

Rendering the attached card through AC 2.9.0, the DOM tree will be the following. Note the convoluted `<img>` with `role="link"`.

```html
<div style="display: flex; align-items: flex-start; justify-content: flex-start; box-sizing: border-box; flex: 0 0 auto;">
  <img class="ac-image ac-selectable x-hidden-focus" tabindex="0" role="link" aria-label="This is title" title="This is title" src="https://www.bing.com/th?id=AMMS_538ac5fe8754dd7df7589f49378971dc" alt="This is alt text" style="min-width: 0px; width: 40px; max-height: 100%; border-radius: 50%; background-position: 50% 50%; background-repeat: no-repeat;">
</div>
```

When reading using a screen reader (Windows Narrator + Edge), it will say: "link this is title". And it will not read the alt text "this is alt text". Even when we press <kbd>CAPS</kbd> + <kbd>DOWN</kbd> to dig into the content of the link, the alt text will not be narrated.

In comparison, when implementing an image link in HTML with the following DOM tree:

```html
<a aria-label="This is title" href="https://bing.com/">
  <img alt="This is alt text" src="https://www.bing.com/th?id=AMMS_538ac5fe8754dd7df7589f49378971dc" />
</a>
```

The screen reader will narrate:

- "Link image this is title"
- Then, press <kbd>CAPS</kbd> + <kbd>DOWN</kbd> to dig into the content of the link
- "This is alt text"

(Related to https://github.com/microsoft/BotFramework-WebChat/issues/4025)

### Screenshots

The card is rendered as follow.

![image](https://user-images.githubusercontent.com/1622400/135167979-1cdbe028-c336-4dd0-9440-5e8303349ef3.png)

### Card JSON

```JSON
{
    "type": "AdaptiveCard",
    "body": [
        {
            "type": "Image",
            "style": "Person",
            "size": "Small",
            "url": "https://www.bing.com/th?id=AMMS_538ac5fe8754dd7df7589f49378971dc",
            "altText": "This is alt text",
            "selectAction": {
                "type": "Action.OpenUrl",
                "title": "This is title",
                "url": "https://github.com/"
            }
        }
    ],
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.3"
}
```

**Comment by cwhitten** [comment]
Hi team - please be aware that this is a Severity 2 Accessibility bug and needs to follow compliance guidelines
